site/pages/docs/versions/rdmp-en.hbs

```diff
@@ -20,12 +20,12 @@
 		</thead>
 		<tbody>
 			<tr>
-				<td><time>2015-08-18</time></td>
-				<td>v4.0.17 release</td>
+				<td><time>2015-09-18</time></td>
+				<td>v4.0.18 release</td>
 			</tr>
 			<tr>
-				<td><time>2015-07-21</time></td>
-				<td>v4.0.16 release</td>
+				<td><time>2015-08-18</time></td>
+				<td>v4.0.17 release</td>
```

Comment:
Those dates are not the right ones, use the following ones instead

https://github.com/wet-boew/wet-boew/milestones